mozilla/bedrock
# Update Reps Council on Leadership page after H2 2022 elections

## One-line summary

This adjusts the Reps Council group on the Leadership page after the latest H2 2022 Reps Council elections. Announcement here: https://discourse.mozilla.org/t/important-council-elections-h2-2022-results-and-next-steps/107323/

## Significant changes and points to review

* 2 people removed, 2 new people added
* New images have been run through ImageOptim

## Screenshots

![image](https://user-images.githubusercontent.com/330324/201524135-3e30051e-d17a-4916-bad9-001e045ba024.png)

To test this work:
* Go to `/en-US/about/leadership/reps-council/`
* Make sure all images show up and links work correctly
